tools/builder/clean-package.js

```diff
@@ -14,11 +17,9 @@ async function run() {
       continue
     }
 
-    console.log(`Removing ${file}`)
+    // console.log(`Removing ${file}`)
```

Comment:
NIT: Did you mean to leave this in here?

Reply:
Yep, this was intentional - someday I should make it a flag, but the `console.log` is _way_ slower than the delete action it's performing. The primary purpose of the log message was debugging, which is less important now that things are stable, but I left the comment in there for easy un-commenting to debug later.